Utils/Dataflow/030_PDFAnalyzer/manager.py

```diff
@@ -123,76 +156,75 @@ def find(self, text, intervals, datasets):
+##                                                "dashes"])
         return (text, datasets)
 
-# This does not works, probably because of differences between "" and r"""""" (re module throws exception).
-##f = open(CATEGORIES_FILE, "r")
-##categories = json.load(f)
-##f.close()
-##dataset_categories = []
-##for c in categories:
-##    regular = ""
-##    for [r, comment] in c["regular"]:
-##        regular += r
-##    dataset_categories.append(DatasetCategory(c["name"], regular))
-
-group = DatasetCategory("group", r"""group                      # Indicates group dataset.
-                                           \n*\.\n*             # Field separator
-                                           [a-zA-Z\d\-:]+       # Group name. Examples: phys-higgs, phys-beauty.
-                                           (\n*[._]\n*[a-zA-Z\d\-:!]+)+
-                                           """)    
-user = DatasetCategory("user", r"""user                         # Indicates user dataset.
-                                           \n*\.\n*             # Field separator
-                                           [a-zA-Z\d\-:]+       # User name.
-                                           (\n*[._]\n*[a-zA-Z\d\-:!]+)+
-                                           """)    
-montecarlo = DatasetCategory("montecarlo", r"""mc\d\d           # Project. Examples: mc08, mc12.
-                                           \n*_\n*              # Field part separator
-                                           [a-zA-Z\d!]+         # Project sub tag. Examples: 7TeV, 1beam, cos.
-                                           \n*\.\n*             # Field separator
-                                           [\dINTERVAL!]+       # DataSet ID(DSID)
-                                           (\n*[._]\n*[a-zA-Z\d\-:!]+)+
-                                           (\n*_\n*[a-z]\d+)+   # AMITag or several
-                                           (_tid\d+(_\d\d)?)?   # Possible production system task and subtask numbers
-                                           """)
-physcont = DatasetCategory("physcont", r"""[a-zA-Z\d\-_\.:!]+
-                                           \n*\.\n*             # Field separator
-                                           PhysCont             # prodStep - it's always the same.
-                                           \n*\.\n*             # Field separator
-                                           [a-zA-Z\d\-_\.:!]+   #
-                                           \n*\.\n*             # Field separator
-                                           [t0proge]+\d\d_v?\d\d# version.
-                                           (\n*_\n*[a-z]\d+)*   # Possible AMITag or several.
+group = DatasetCategory("group", r"""
+group              # Indicates group dataset.
+\n*\.\n*           # Field separator
+[a-zA-Z\d\-:]+     # Group name. Examples: phys-higgs, phys-beauty.
+(\n*[._]\n*[a-zA-Z\d\-:!]+)+
+                                     """)    

```

Comment:
Seems like you have the trailing spaces here and in line [183](https://github.com/PanDAWMS/dkb/pull/39/commits/a99989c2cd6402df9d03a22c12ca22b3185d3dd9#diff-ac4be2bc1e072348e27b33ba39ff0b31R183)